setup.cfg

```diff
@@ -31,6 +31,7 @@ install_requires =
     openpyxl
     pandas>1.2.0
     pandas_path
+    protobuf==3.20.1
```

Comment:
For context, see this [github action error](https://github.com/drivendataorg/zamba/runs/6680469946?check_suite_focus=true#step:7:59) and [protocol buffers update](https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates)